ui/desktop/src/components/BaseChat.tsx

```diff
@@ -242,7 +229,7 @@ function BaseChatContent({
   const { createNewSessionIfNeeded } = useSessionContinuation({
     chat,
     setChat,
-    summarizedThread,
+    summarizedThread: [],
```

Comment:
@zanesq I wanted to ask you about this one. Can you clarify why the `useSessionContinuation` hook needed the summarized thread before? Likely need to make some kind of change here before going ahead.